src/Tizen.NUI/src/public/BaseComponents/TextUtils.cs

```diff
@@ -1819,8 +1817,7 @@ public static string CopyToClipboard(TextField textField)
         /// </summary>
         /// <param name="textEditor">The textEditor control from which the text is cut.</param>
         /// <returns>The cut text.</returns>
-        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
the "... and ... and ..." doesn't sound right, maybe write it "Cut the previously selected text from the text control and **copy it** into the clipboard.", the returned value is described with returns tag and this is enough.

Reply:
@wiertel , 
Done.